packs/data/pfs-season-1-bestiary.db/palace-guard-pfs-1-17.json

```diff
@@ -1055,7 +1055,7 @@
             "type": "lore"
         }
     ],
-    "name": "Palace Guard (1-17)",
+    "name": "Palace Guard (PFS 1-17)",
```

Comment:
Difference from Bestiary version:
Level 
Fey trait
Mwangi and Sylvan languages
AC
HP
Fort and Will saves
Remove Royal Defender
Add crossbow with bolts and sap